source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -4,11 +4,69 @@
 Release notes - Mainnet
 =======================
 
-.. contents::
-   :local:
-   :backlinks: none
+Mainnet 4: Sirius
+=================
+
+June 15, 2022
+
+Concordium Node 4.1.1
+---------------------
+
+Concordium Node 4.1.1 introduces new functionality to support delegation to baker pools or passive delegation, and a new version Smart Contracts.
+
+Note that when the protocol update happens on June 23, 2022 that the cool-down period for reducing baker stake or stopping baking increases from one week to two weeks. If you reduce your stake or stop baking BEFORE the protocol update takes effect, the cool-down remains one week.
```

Comment:
Baker (pool owner) cooldown is set for 3 weeks. So it increases from one week to 3 weeks.
Delegator cooldown is set to 2 weeks.

Reply:
Fixed.